instances/treasury-devdao.near/widget/lib/common.jsx

```diff
@@ -39,14 +44,14 @@ function getApproversAndThreshold(treasuryDaoID, kind) {
         denominator += value;
       }
     });
-  } else {
-    numerator = 1;
-    denominator = 2;
   }
+  const approverAccounts = Array.from(new Set(approversGroup));
 
   return {
-    approverAccounts: Array.from(new Set(approversGroup)),
-    threshold: numerator / denominator,
+    approverAccounts,
+    requiredVotes:
+      requiredVotes ||
+      Math.floor((numerator / denominator) * approverAccounts.length) + 1,
```

Comment:
What is the reason the required votes is different from the contract response?

Reply:
sorry I don't understand the question, different from contract meaning?
we don't get the exact number from contract, we only get the threshold (which can be a ratio or an absolute number), when it's a ratio we need to first calculate the no of people allowed to vote on that particular proposal type (i.e `approverAccounts`) and then get the absolute number, but in case it's already an absolute number we use that

Reply:
I see, that makes sense. Thanks for the explanation.